database/src/migrations/release.0.8.0/fn_calculate_area_intersect.sql

```diff
@@ -0,0 +1,17 @@
+-- *******************************************************************
+-- Procedure: fn_calculate_area_intersect
+-- Purpose: returns true if both geometries intersect by the given tolerance amount (0.4 == 40% coverage)
+--
+-- MODIFICATION HISTORY
+-- Person           Date        Comments
+-- ---------------- ----------- --------------------------------------
+-- alfred.rosenthal@quartech.com
+--                  2024-01-15  initial release
+-- *******************************************************************
+CREATE OR REPLACE FUNCTION fn_calculate_area_intersect(geom1 geometry, geom2 geometry, tolerance float)
+RETURNS boolean AS $$
+BEGIN
+  RETURN ST_Area(ST_Intersection(geom1, geom2)) / ST_Area(geom1) >= tolerance
```

Comment:
Might be beneficial to initialize a variabe with the output of "ST_Area(ST_Intersection(geom1, geom2))" and reference on both sides of the OR statement.

Only if ST_Area function is somewhat slow / complex.

Reply:
yeah good call, I've updated the script to so the area and intersection is only run once